programs/client/Client.cpp

```diff
@@ -243,6 +246,17 @@ class Client : public Poco::Util::Application
         /// Set path for format schema files
         if (config().has("format_schema_path"))
             context.setFormatSchemaPath(Poco::Path(config().getString("format_schema_path")).toString());
+
+        /// Initialize query_id_formats if any
+        if (config().has("query_id_formats"))
+        {
+            Poco::Util::AbstractConfiguration::Keys keys;
+            config().keys("query_id_formats", keys);
+            for (const auto & name : keys)
+                query_id_formats.emplace_back(name + ":\n", config().getString("query_id_formats." + name));
+        }
+        if (query_id_formats.empty())
+            query_id_formats.emplace_back("Query id:\n", "{query_id}");
```

Comment:
Why need \n?

Reply:
URL hinters usually prefers one-line strings. Although some terminals can correctly understand line wraps, it won't work for screen or tmux. With `\n` it's more likely to generate one-line urls.

Reply:
I still consider "speedscope" as minor use case in contrast to the default behaviour when client will just display something like
```
Query id: 01db8425-415a-4ec2-bcbf-14dbc49aa8dd
```
for every query.